sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+### Modes of Interdisciplinarity
+
+Disciplines carry with them a specific methodological culture, a set of practices that serve to define and protect the boundaries of the field as an object and system of practice. The problem with disciplines, for me, is partially one of positionality. I have never felt entirely at home in any of them. Though it is tempting (and not altogether inaccurate) to refer myself as an "interdisciplinary artist," or to argue that I am drawing together a collection research methods from each of the aforementioned disciplines in a manner that is consistent with "an interdisciplinary practice," several concerns emerge for me with such a construction. Despite its casual and prolific use throughout arts and design scholarship, the scope and definition of interdisciplinarity as a concrete set of methods or practices is impossibly vague. More importantly, the concept carries with it strong connotations of unification or blending that, for me, risk masking the particularities of my thinking and creative practices in each discipline.
```

Comment:
Comment from David Cecchetto by email: "In the modes section, I think it is worth specifying just what it means to not be at home in a discipline. What are the specific constraints, how are they enacted (and how are these enactments different throughout the process), and what do they foreclose? I’m not sure you need to answer these all, but something to get a sense of your particular style of ‘discomfort’ would be helpful if you’re going to negatively define INTD as much as affirm it. You get at this by working through the modes, but I wonder if there isn’t a simple and pragmatic example of an utterance that would go unheard in one discipline but not in another (for example, the silence of your videos becomes rhetorical—i.e. “silence”—when listened to as music, but this isn’t reversible…)"